browser/src/control/Control.Tabs.js

```diff
@@ -95,32 +95,27 @@ L.Control.Tabs = L.Control.extend({
 				callback: (this._moveSheetRight).bind(this),
 				visible: areTabsMultiple
 			},
-		};
-
-		if (!window.mode.isMobile()) {
-
-			// no blacklisting available for this context menu so only add when needed
-			this._menuItem['.uno:Move'] = {
+			'.uno:Move': {
 				name: _UNO('.uno:Move', 'spreadsheet', true),
 				callback: (this._moveOrCopySheet).bind(this),
 				visible: areTabsMultiple
-			};
-
-			this._menuItem['.uno:CopyTab'] = {
+			},
+			'.uno:CopyTab': {
 				name: _('Copy Sheet...'),
 				callback: function() {this._map.sendUnoCommand('.uno:Move');}.bind(this),
 				visible: function() {
 					return !areTabsMultiple();
 				}
-			};
-			    if (!this._map.isReadOnlyMode() || window.mode.isTablet()) {
-				L.installContextMenu({
-					selector: '.spreadsheet-tab',
-					className: 'cool-font',
-					items: this._menuItem,
-					zIndex: 1000
-				});
-			}
+			},
+		};
+
+		if ((!window.mode.isMobile() && !this._map.isReadOnlyMode()) || window.mode.isTablet()) {
```

Comment:
on tablet do we want to add the menu in readonly?

Reply:
It seemed a little weird to me too, but I applied the principle of Chesterton's fence: that is, don't break it unless you understand it.

The condition before was
```js
if (!window.mode.isMobile()) {
	if (!this._map.isReadOnlyMode() || window.mode.isTablet()) {
```

and the condition after was
```
if ((!window.mode.isMobile() && !this._map.isReadOnlyMode()) || window.mode.isTablet()) {
```

given that $window.mode.isTablet() \implies !window.mode.isMobile()$, these are the same condition

Reply:
I can take a deeper look into why it was this way if you'd like

Reply:
Amusingly enough git blame suggests this came from https://github.com/CollaboraOnline/online/pull/7301 :-)

"Show tabs context menu on a tablet's press" because "tablets no longer had the context menu registered as they always start in read only mode"

So I guess its fine to retain that fix

Reply:
it feels to me like there should be a better way to do that that registers/unregisters the context menu at the right time ... but I guess it makes sense, desktop doesn't switch from read-only to not whereas tablet does

Reply:
well, certainly we can keep it for now :)